multi_validator_harness_test.go

```
@@ -0,0 +1,445 @@
+	if err := validateLedgerGenesis(app.appCodec, state); err != nil {
+		t.Fatalf("multi-validator genesis is not exactly bank-backed: %v", err)
+	}
+}
+
+func TestMultiValidatorConsensusRecovery(t *testing.T) {
+	if os.Getenv(multiValidatorSmokeEnv) != "1" {
+		t.Skipf("set %s=1 to run the multi-validator process harness", multiValidatorSmokeEnv)
+	}
+
+	binary := filepath.Join(t.TempDir(), "truerepublicd")
+	build := exec.Command("go", "build", "-o", binary, ".")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)